lighthouse-core/test/gather/gatherers/dobetterweb/optimized-images-test.js

```diff
@@ -22,49 +22,49 @@ const traceData = {
     {
       _url: 'http://google.com/image.jpg',
       _mimeType: 'image/jpeg',
-      _resourceSize: 10,
+      _resourceSize: 10000,
```

Comment:
what's the deal with this change? Just more realistic sizes/getting above the 4k cutoff?